index.html

```diff
@@ -203,6 +204,21 @@
         refresh_grid_lines();
       }
 
+      function subdivide_grid(operation) {
+        var drawing = document.getElementById("drawing");
+        if (operation === "1") {
+          grid_size = 50;
+        } else if (operation === "2") {
+          grid_size = 50 / 2;
+        } else if (operation === "3") {
+          grid_size = 50 / 3;
+        } else if (operation === "4") {
+          grid_size = 50 / 4;
```

Comment:
Notice the pattern? Instead of all these ifs, you could convert operation/subdivisions to a number, and do `50 / subdivisions`